### DISCUSSION TITLE
Vanilla Audio/Vanilla Video default setting options

### DISCUSSION THREAD

**Discussion by Qo1697** [AprilSylph/XKit-Rewritten#1314] [Suggestion Box]
The 'Vanilla Audio' and 'Vanilla Video' features both let you change the default volume that video/audio is set to, which is excellent.
However, the only options given are 25%/50%/75%/100%.
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/20309453/2f043ff9-7ed9-4b8b-bc3e-9b9db05d952c)

The smallest of these is still too loud for me on my current setup. Being able to enter a custom value there would be ideal, but if not I'd appreciate having 10% as an option on the dropdown.

**Comment by ConnorTheSilly** [comment]
I KNOW THIS IS OLD BUT PLEASE I NEED THIS
